docs/quality-assurance/safety-assurance.md

```diff
@@ -0,0 +1,98 @@
+* Speak up, if you feel that safety is being compmised in some way, raise it!!!!!.
+* Exercise judgement, not the same as going out on a limb but exercising common sense
+* ALARP – As low as reasonably practicable
+
+# What does this mean for delivery teans
+
+We need to consider  the safey and the potential safety implications of errors and/or delays to the products that we are working on, as well as how they will be maintained going forward. 
+So we actually consider safety all the time, we just probably don't really think about it in that way. This provides some guidelines to think about when planning what type of tests you may wish to design for a chamge.
+
+Safety assurance will be modelled on the threat modelling process that we already follow for security risks, in this case it is safety risks.
+
+
+
+# Prior to sprints
+
+* A high level review of the product, the aim to identify the key areas that could cause errors and delay
+* The test approach document to include how safety issues will be asseed (Much like security issues are)
+* What is the expected SLA for the product
+* What recovery time is expected
+* How will the product be maintained/updated going forward minimising any safety impacts
+
+The aim of considering these at a high level initially will help potentially the technology to be used, the building of the code as well as the types of tests that my be required.
+
+
+## Each Sprint
+
+* A Safety Modelling Workshop (This can be combined with the current threat modelling workshop) This could be simply an exercise on a 
+  PBI by PBI basis of considering any safety case implications). Any PBI that is considered to have a safety case implication to be
+  tagged with safety. This should help drive :
+* What if PBI's, beside the happy path acceptance criteria , testing for instance that if the system fails it can be recovered within
+* This may and should create new PBI's that are effectiveky negative requirements i.e what if this fails,can it be recovered in time that does not imoact safety (error or delay)  a time period that does not impact the mariner should also form part of testing phase. 
+* Tag any PBI's that have a safety implication with a tag
+* Against PBI's with a tag of safety , the decesions around the consideration and decesions are documented and agreed.
+* All requirements have acceptance criteria
+* All acceptance criteria have a test(s) linked to them
+* All deployed code is linked back to relevant tests
+* All requirements (PBI’s) have acceptance criteria
+* There is a regression pack (If applicable)
+* An auditable history of test runs (Automated and manual) is kept
+* We have run a risk review so that the functionality with the largest safety impact is considered and potential priority (This is 
+  something that the product owner would sign off on)
+
+
+## Prior to go live
+
+* Is the system compliant and certified
```

Comment:
How is this achieved? Certified? When and by who? All the time? This is prompting more questions rather than answering them